network-api/networkapi/management/commands/generate_mozfest_site.py

```diff
@@ -0,0 +1,37 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from wagtail.core.models import Site as WagtailSite
+from networkapi.mozfest.models import MozfestPrimaryPage
+import networkapi.mozfest.factory as mozfest_factory
+import random
+
+
+class Command(BaseCommand):
+    help = 'Generate a site and content for the Mozfest sub-site.'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--seed',
+            action='store',
+            dest='seed',
+            help='A seed value to pass to Faker before generating data',
+        )
+
+    def handle(self, *args, **options):
+
+        if options['seed']:
+            seed = options['seed']
+        elif settings.RANDOM_SEED is not None:
+            seed = settings.RANDOM_SEED
+        else:
+            seed = random.randint(0, 5000000)
+
+        print('Dropping all Mozfest pages')
+        MozfestPrimaryPage.objects.all().delete()
```

Comment:
What about MozfestHomepage?

Reply:
There might be other page types too, is there a way to say "Homepage and all descendants"?

Reply:
Unfortunately all I think I can do is filter on the immediate parent. If we end up adding new page types we can just call the same methods against the new Page class.

Reply:
I guess I'm confused as to why this script drops anything, instead of just being for generating the site itself?

Reply:
Here's a thought, if we've got pages on production now, then we don't need to generate any new ones after a copy. We need to only update the hostname of the Site.

If that's true, then we can probably just delete this command.

Reply:
But it's useful to generate them for staging/dev. Are you just referring to deleting this line?

Reply:
As it is now, the load_fake_data management command will generate the mozfest pages locally. Since we added a Mozfest site and pages to prod, those will be copied over on the weekly clone, making this command unnecessary, I think.

Reply:
Great.

Reply:
`MozfestHomepage` inherits from `MozfestPrimaryPage`, so that give me the nice benefit of being able to query for the whole lot of Pages through `MozfestPrimaryPage`